docs/pages/api-docs/year-picker.json

```diff
@@ -4,12 +4,12 @@
     "shouldDisableYear": { "type": { "name": "func" } }
   },
   "name": "YearPicker",
-  "styles": { "classes": ["root"], "globalClasses": {}, "name": "MuiYearPicker" },
+  "styles": { "classes": [], "globalClasses": {}, "name": null },
```

Comment:
Not sure how to bring the "root" and "name" back 🤔 @oliviertassinari @mnajdova

Reply:
I have fixed this in 2058f10. @eps1lon is working on reversing the approach in #25754